Dockerfile.windows

```diff
@@ -1,27 +1,29 @@
 # escape=`
 
 ARG TAG=ltsc2019
-ARG AWS_FOR_FLUENT_BIT_VERSION
 
 #
 # Runtime Image - Windows Server Core
 #
 FROM mcr.microsoft.com/windows/servercore:$TAG as runtime
 
+ARG AWS_FOR_FLUENT_BIT_VERSION
+
 WORKDIR C:/
 
 ADD fluent-bit /fluent-bit
 
 ADD fluent-bit-plugins /fluent-bit
 
-ADD ecs /ecs
+ADD ecs_windows /ecs_windows
 
 ADD certs /certs
 
-ADD AWS_FOR_FLUENT_BIT_VERSION /AWS_FOR_FLUENT_BIT_VERSION
-
 ADD entrypoint.ps1 /entrypoint.ps1
 
+# Add the version file.
+RUN Powershell -Command New-Item -Path C:/ -Name AWS_FOR_FLUENT_BIT_VERSION -ItemType "file" -Value $env:AWS_FOR_FLUENT_BIT_VERSION
+
```

Comment:
nit: I think this should be a separate commit

Reply:
rectified the same in the latest revision. The commits have been segregated.